lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
+    TestTrack::FakeServer.reset!(RSpec.configuration.seed)
```

Comment:
do we need this anymore? we're using a stub in the helper above which will be automatically undone at the end of each spec. and the split registry is sourced from the schema file and shouldn't be mutated at all during test runs, right?